.chloggen/nodejs-featuregate.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: pkg/instrumentation
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add python instrumentation capability behind a feature gate which is enabled by default.
```

Comment:
@TylerHelmuth could you change the changelog note here :)

Reply:
I'm on PTO but if you hit me with a Suggestion I'll commit it.